src/OSX/Avalonia.MonoMac/MonoMacPlatform.cs

```diff
@@ -87,8 +122,10 @@ namespace Avalonia
 {
     public static class MonoMacPlatformExtensions
     {
-        public static T UseMonoMac<T>(this T builder) where T : AppBuilderBase<T>, new()
+        public static T UseMonoMac<T>(this T builder, bool? useDeferredRendering = null) where T : AppBuilderBase<T>, new()
         {
+            if (useDeferredRendering.HasValue)
+                MonoMac.MonoMacPlatform.UseDeferredRendering = useDeferredRendering.Value;
             return builder.UseWindowingSubsystem(MonoMac.MonoMacPlatform.Initialize);
```

Comment:
Any chance we can add the platform name as the second parameter here? That will help once we bring MonoMac support back onto .NET Framework/Mono runtime since it will correctly enable the desktop version of `UsePlatformDetect`.

Reply:
I thought that `UsePlatformDetect` is using attributes for platform discovery:

https://github.com/AvaloniaUI/Avalonia/blob/master/src/OSX/Avalonia.MonoMac/AssemblyInfo.cs#L5

Reply:
Yep, it seems to be doing that: https://github.com/AvaloniaUI/Avalonia/blob/master/src/OSX/Avalonia.MonoMac/AssemblyInfo.cs#L5t

Do I need to change something else?

Reply:
I was mistaken. We need the second parameter for MonoMac-specific AvaloniaModule support when not using `UsePlatformDetect`.